target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -39,6 +40,10 @@ case "$FIRMWARE" in
 		caldata_extract "art" 0x1000 0x440
 		ath9k_patch_mac $(mtd_get_mac_ascii u-boot-env ethaddr)
 		;;
+	mikrotik,rb-wapg-5hact2hnd)
+		mikrotik_caldata_extract "hard_config" 0x1000 0x440 $FIRMWARE
```

Comment:
For all variants of the "normal" caldata extraction, we hard-coded $FIRMWARE in the function.
So, I'd follow this style here, too, and remove the argument for the firmware.

Despite, consider using "art" for the label here and in DTS, so we have a little bit more consistency there between devices.

Reply:
Cool.